### ISSUE TITLE
`astro serviceaccount create` now fails with Role required error from API

### ISSUE THREAD

**Issue by ashb** [astronomer/astro-cli#278]
```
$ astro sa cr -d cjzshq8sr00gx0b03bjg7bd4l --label wh-data-us-sdp-us-east-1-nonprod-service-account-cicd
Error: API error (400): {"errors":[{"message":"Field \"createServiceAccount\" argument \"role\" of type \"Role!\" is required but not provided.","locations":[{"line":8,"column":4}],"extensions":{"code":"GRAPHQL_VALIDATION_FAILED"},"level":"ERROR","timestamp":"2019-08-26T14:30:28"}]}
```

Since we changed the houston API to now require the `Role` argument to the "createServiceAccount" mutation the CLI is now unable to create service accounts.  If the role is not specified it should default to the Workspace/Deployment Editor role as appropriate.

**Comment by ashb** [comment]
Was thinking this was a quick fix in a 0.10.1, but we could just do the larger one, yes.

**Comment by andriisoldatenko** [comment]
@schnie are you OK if i quick fix it in 0.10.1?

**Comment by schnie** [comment]
Yea lets do it
